src/userdb/userdbctl.c

```diff
@@ -123,7 +121,6 @@ static int show_user(UserRecord *ur, Table *table, JsonVariant **array) {
                         return table_log_add_error(r);
 
                 break;
-        }
```

Comment:
Yes, `d` would be visible further down. But I think that's OK. This is like any other variable that we define somewhere in the function, maybe use once or twice, and then it remains available until the end of the scope. Sometimes we add a separate block just to limit scope of variables, but I don't see any special justification for this here.

Reply:
No, it's not like that at all, since the variable exists, but the initialization didn't happen in those code blocks further down. it's a really weird netherworld if semi-existence, where the variable exists, but it's not initialized.

Reply:
But we have tens of thousands of uninitialized variables… I push for RAII, because I think it's easier to read and helps to avoid errors, but it's still very much the exception in our codebase. Those two variables are not any different. They are initialized in the paths that use them, and remain uninitialized otherwise.